conf/schema.yml

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
How about keeping this in `lib/Reek/Configuration`, similar to where we store the HTML template?